spec/rspec/mocks/diffing_spec.rb

```diff
@@ -83,6 +83,21 @@
       end
     end
 
+    if RUBY_VERSION >= "3" && RSpec::Support::RubyFeatures.kw_args_supported?
```

Comment:
I added the `RUBY_VERSION >= "3"` which should solve the CI issues.

Reply:
We typically add such things to `rspec-support` and use a self-explanatory method on `RubyFeatures`. Do you think it's possible to come up with a name that would explain why a special case is needed?

Reply:
Hum, `RubyFeatures.has_ruby2_compatibility_keywords`?

Reply:
Sure. Should I make a PR to `rspec-support`?

Reply:
Yes, please :+1: